.goreleaser.yml

```diff
@@ -25,6 +25,33 @@ archives:
       - "README.md"
       - "LICENSE"
       - "completions/*"
+    format_overrides:
+      - goos: "windows"
+        format: "zip"
+chocolateys:
+  - name: "spicedb"
+    package_source_url: "https://github.com/authzed/spicedb"
+    owners: "AuthZed, Inc"
+    title: "SpiceDB"
+    project_url: "https://github.com/authzed/spicedb"
+    use: "archive"
+    url_template: "https://github.com/authzed/spicedb/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
+    icon_url: "https://authzed.com/favicon.svg"
+    copyright: "2024 AuthZed, Inc"
+    license_url: "https://github.com/authzed/spicedb/blob/main/LICENSE"
+    require_license_acceptance: false
+    project_source_url: "https://github.com/authzed/spicedb"
+    docs_url: "https://authzed.com/docs"
+    bug_tracker_url: "https://github.com/authzed/spicedb/issues"
+    tags: "spicedb zanzibar authz rebac rbac abac fga"
+    summary: "SpiceDB is a graph database purpose-built for storing and evaluating access control data."
+    description: |
+      Open Source, Google Zanzibar-inspired permissions database to enable fine-grained access control for customer applications
+    release_notes: "https://github.com/authzed/spicedb/releases/tag/v{{ .Version }}"
+    api_key: "{{ .Env.CHOCOLATEY_API_KEY }}"
+    source_repo: "https://push.chocolatey.org/"
+    skip_publish: false
+    goamd64: "v1"
```

Comment:
let's remove any properties that are the default for better signal/noise ratio:
- goamd64
- skip_publish
- require_license_acceptance